package.json

```diff
@@ -1,18 +1,17 @@
 {
-  "name": "taco-simulate-server",
-  "version": "0.1.3",
-  "description": "Server and app host module for taco-simulate (Cordova plugin simulation).",
-  "main": "src/server.js",
+  "name": "cordova-simulate",
+  "version": "0.2.0",
```

Comment:
Generally don't bump the version until we actually do a release. Otherwise people who install from the repo will be at the same version as the release when it finally comes out. Either leave this as `0.1.3` until we release, or do what we do with Cordova and make it something like `0.2.0-dev` (which is treated as a lower version than `0.2.0`) until we release.